### ISSUE TITLE
Comment added to value of key

### ISSUE THREAD

**Issue by Tohjuler** [mikigal/ConfigAPI#11]
Hi again, i found a bug where i added a comment to a key, and the command was added to the value.

Code 
```java
@Comment("The message that will be sent to the player, if their message is filtered")
    default String getFilteredMessage() {
        return "&c&lCHAT &8| &7Din besked indeholder ord, der ikke er tilladt, og er blevet rettet.";
    }
```
    
The config
```yaml
filtered_message: '&c&lCHAT &8| &7Din besked indeholder ord, der ikke er tilladt, # The message that will be sent to the player, if their message is filtered
  og er blevet rettet.'
```    
Just ask if you need any other info :D

**Comment by mikigal** [comment]
Please try using both comment styles - `CommentStyle.INLINE` and `CommentStyle.ABOVE_CONTENT`.
You have to specify it when initializing API
Let me know does issue exist in both cases

**Comment by Tohjuler** [comment]
When using ``CommentStyle.ABOVE_CONTENT`` the comment is at the right place but the value is broken into 2 lines.

![image](https://github.com/mikigal/ConfigAPI/assets/84204734/1e0a4f85-e9b8-4a3b-8a46-c4dde9207ebe)

**Comment by mikigal** [comment]
Hi, 
problem was related to splitting field into few lines if it's length is above 80 characters.
Comment was added in the first line so it caused adding it into String

Fixed in https://github.com/mikigal/ConfigAPI/commit/7f11bc91cb1a3b3590fb5b6d0c9a4cfc81d6d315 (released in v1.2.5)
